### PR TITLE
add note about DB name in connection url with mongodb atlas

### LINKED ISSUE #5562: Developer freindly Doc for solving Error when running prisma+mongodb with connection string with no database name
### Bug description

HI, Im integrating prisma & mongodb and got a problem when following the documentation [here](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/mongodb/connect-your-database-typescript-mongodb), for more developer friendly experience doc, it would be helpful to update it, see the image included for how

### How to reproduce

1. Go to Doc guide for set up prisma + mognodb, [here](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/mongodb/connect-your-database-typescript-mongodb)
2. Follow the step-by-step guide for integrating Prisma into your project.
3. Start by creating a basic database cluster on MongoDB.
4. Access the Atlas dashboard for MongoDB management.
5. Navigate to the 'Database' tab.
6. Establish a connection to the database using the provided drivers.
7. Incorporate the database connection string into your application's code.
8. prisma_doc Note that MongoDB Atlas generates an environment link which, crucially, lacks the database name.  generate env link without the Database name
9. Commonly, users copy the provided MongoDB environment key directly into their project's .env file, assuming it is complete.
10. The error message displayed is: "
```
Error: 
Invalid prisma.blog.findMany() invocation: Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.
```

### Expected behavior

To enhance the developer-friendly aspect of prisma documentation, it would be beneficial to add a concise, highlighted notice, perhaps with an exclamation mark, advising developers to include the database name when copying the environment variable for the MongoDB URL link. This addition would significantly improve the clarity and usability of prisma documentation, see image for where the information could be placed:
prisma_doc_2

### Prisma information

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}
...

### Environment & setup

- OS: Windows, win32, x64
- 
- Database: MongoDB 
- Node.js version: v18.18.2

### Prisma Version

  Prsima: 5.7.1

### PR DESCRIPTION
**Pull request by nikolasburk**
fixes #5562

### CODE REVIEW

**Review by jharrell — DISMISSED** [review]

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-node-mongodb.mdx [hunk 1]
```diff
@@ -41,4 +41,14 @@ Here's a short explanation of each component:
 - `PASSWORD`: The password for your database user
 - `HOST`: The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) (or [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos)) instance is running
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
-- `DATABASE`: The name of the database
+- `DATABASE`: The name of the database. Note that if you're using MongoDB Atlas, you need to manually append the database name to the connection URL because the environment link from MongoDB Atlas doesn't contain it.
+
+## Troubleshooting
+
+### Error in connector: SCRAM failure: Authentication failed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-node-mongodb.mdx]: Should it be in a quoted block as it's an error message?

```suggestion
### `Error in connector: SCRAM failure: Authentication failed.`
```

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-node-mongodb.mdx [hunk 2]
```diff
@@ -41,4 +41,14 @@ Here's a short explanation of each component:
...
+
+If you see the `Error in connector: SCRAM failure: Authentication failed.` error message, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+
+### Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-node-mongodb.mdx]:
```suggestion
### `Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.`
```

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 3]
```diff
@@ -39,6 +39,14 @@ Here's a short explanation of each component:
 - `PASSWORD`: The password for your database user
 - `HOST`: The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) (or [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos)) instance is running
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
-- `DATABASE`: The name of the database
+- `DATABASE`: The name of the database. Note that if you're using MongoDB Atlas, you need to manually append the database name to the connection URL because the environment link from MongoDB Atlas doesn't contain it.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx:42]: Can we add an example database connection string here?

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 4]
```diff
@@ -39,6 +39,14 @@ Here's a short explanation of each component:
...
 
-If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by adding `?authSource=admin` to the end of the connection string. See [this GitHub discussion](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) for additional details.
\ No newline at end of file
+## Troubleshooting
+
+### Error in connector: SCRAM failure: Authentication failed.
+
+If you see the `Error in connector: SCRAM failure: Authentication failed.` error message, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+
+### Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]:
```suggestion
### `Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.`
```

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 5]
```diff
@@ -39,6 +39,14 @@ Here's a short explanation of each component:
...
 - `HOST`: The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) (or [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos)) instance is running
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
-- `DATABASE`: The name of the database
+- `DATABASE`: The name of the database. Note that if you're using MongoDB Atlas, you need to manually append the database name to the connection URL because the environment link from MongoDB Atlas doesn't contain it.
 
-If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by adding `?authSource=admin` to the end of the connection string. See [this GitHub discussion](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) for additional details.
\ No newline at end of file
+## Troubleshooting
+
+### Error in connector: SCRAM failure: Authentication failed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]:
```suggestion
### `Error in connector: SCRAM failure: Authentication failed.`
```

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-node-mongodb.mdx [hunk 6]
```diff
@@ -43,6 +43,14 @@ Here's a short explanation of each component:
...
+- `DATABASE`: The name of the database. Note that if you're using MongoDB Atlas, you need to manually append the database name to the connection URL because the environment link from MongoDB Atlas doesn't contain it.
 
-> If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+## Troubleshooting
+
+### Error in connector: SCRAM failure: Authentication failed.
+
+If you see the `Error in connector: SCRAM failure: Authentication failed.` error message, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+
+### Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-node-mongodb.mdx]:
```suggestion
### `Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.`
```

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 7]
```diff
@@ -41,7 +41,14 @@ Here's a short explanation of each component:
 - `PASSWORD`: The password for your database user
 - `HOST`: The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) (or [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos)) instance is running
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
-- `DATABASE`: The name of the database
+- `DATABASE`: The name of the database. Note that if you're using MongoDB Atlas, you need to manually append the database name to the connection URL because the environment link from MongoDB Atlas doesn't contain it.
 
-> If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+## Troubleshooting
 
+### Error in connector: SCRAM failure: Authentication failed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]:
```suggestion
### `Error in connector: SCRAM failure: Authentication failed.`
```

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 8]
```diff
@@ -41,7 +41,14 @@ Here's a short explanation of each component:
...
+
+If you see the `Error in connector: SCRAM failure: Authentication failed.` error message, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
+
+### Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]:
```suggestion
### `Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed.`
```

**Review by ankur-arch — APPROVED** [review]

**Pull request merged**